template/usage.md

```diff
@@ -0,0 +1,55 @@
+# How to use the Ragger test framework
+
+This framework allows testing the application on the Speculos emulator or on a real device using LedgerComm or LedgerWallet
+
+
+## Quickly get started with Ragger and Speculos
+
+### Install ragger and dependencies
+
+```
+pip install --extra-index-url https://test.pypi.org/simple/ -r requirements.txt
+sudo apt-get update && sudo apt-get install qemu-user-static
+```
+
+### Compile the application
+
+For all required devices, compile the application by running the following command in the container `ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-lite:latest`
+```
+make BOLOS_SDK=$<device>_SDK
```

Comment:
You added more details here: https://github.com/LedgerHQ/app-elrond/pull/19/files
Do you think these should be added on every repo?
Might not be as some are more related to building and sideloading more than to testing. Hence theses instructions might more be located at the root of the repo instead of in the test folder?